h/static/scripts/login-forms/components/SignupForm.tsx

```diff
@@ -9,10 +9,52 @@ import TextField from '../../forms-common/components/TextField';
 import { useFormValue } from '../../forms-common/form-value';
 import { Config } from '../config';
 import type { SignupConfigObject } from '../config';
+import ORCIDIcon from './ORCIDIcon';
 
-export default function SignupForm() {
+type IdProviderBadgeProps = {
+  provider: 'orcid';
+  identity: string;
+};
+
+/**
+ * Box indicating the OIDC / social login identity being associated with a
+ * Hypothesis account.
+ */
+function IdProviderBadge({ provider, identity }: IdProviderBadgeProps) {
+  return (
+    <div data-testid="id-badge">
+      {provider === 'orcid' && (
+        <a href="https://orcid.org" target="_blank" rel="noreferrer">
+          <ORCIDIcon className="inline" aria-label="ORCID icon" />
+        </a>
+      )}{' '}
+      connected: <b data-testid="connected-id">{identity}</b>
+    </div>
+  );
+}
```

Comment:
The design of the badge is not final. I want to play around with some options once we have both Google and ORCID implemented.